### DISCUSSION TITLE
Is this project far right politically? Re: slogan "Make XMPP great again"

### DISCUSSION THREAD

**Discussion by krc** [jsxc/jsxc#1129] [General]
The authors seem to be German, so I presume not MAGA per se, but are they far right wing? I genuinely cannot think of another reason why anyone would pick a slogan based on Trump's. Maybe if someone had been using it since before that election, but I don't see evidence of that here. If they are not right wing and used that slogan because... IDK they think it's funny or something... they must know it is turning away potential users and contributors.

**Comment by sualko** [comment]
Do you feel the Burn?

"Make XMPP great again" was the goal of this project and an ironic jab to US politics. That's it.
